client/src/App.js

```diff
@@ -3,27 +3,31 @@ import FloatingActionButton from './components/FloatingActionButton'
 import SearchBooks from './components/SearchBooks'
 import styled from 'styled-components/macro'
 import useBookData from './hooks/useBookData'
+import BookDetail from './components/BookDetail'
 
 export default function App() {
   const {
     addBook,
+    book,
+    bookDetailModal,
     deleteBook,
     handleChange,
     handleSubmit,
     results,
     searchBooksModal,
     selectedBooks,
     setResult,
+    showDetail,
     toggleSearchBooksModal,
   } = useBookData()
 
   return (
     <StyledApp>
```

Comment:
If you want to use "Styled" in your name, I'd put it at the end. So that it becomes clear more quickly what element it is.